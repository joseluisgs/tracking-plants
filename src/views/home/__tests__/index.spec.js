// Para hacer una copia del componente montado
import { shallowMount } from '@vue/test-utils';
import HomeView from '../index.vue';

describe('Test Home View', () => {
  // Nos devuelve un shallowmont del componente
  // es decir una copia de lo que se renderizarÃ­a en la web
  const build = () => {
    const wrapper = shallowMount(HomeView);
    return {
      wrapper,
    };
  };

  //
  test('should match snapshot', () => {
    const { wrapper } = build();
    // El elemento debe coicidir con la copia montada
    expect(wrapper.element).toMatchSnapshot();
  });
});
