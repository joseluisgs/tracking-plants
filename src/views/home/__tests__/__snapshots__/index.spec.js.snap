// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home View Deber√≠a ser igual a su Snapshot 1`] = `
<div
  class="home"
>
  <listplants-stub
    plants=""
  />
</div>
`;
